src/windows/list.cpp

```diff
@@ -226,11 +227,11 @@ int32_t GetDeviceNumber(HANDLE hDevice) {
   if (result) {
     // printf("[INFO] DeviceNumber: %i\n", deviceNumber.DeviceNumber);
     diskNumber = deviceNumber.DeviceNumber;
-  } else {
-    errorCode = GetLastError();
-    // printf("[INFO] STORAGE_GET_DEVICE_NUMBER: Error 0x%08lX\n", errorCode);
   }
 
+  // errorCode = GetLastError();
```

Comment:
Even though this debugging code has been commented out, shouldn't it still be within the `else` block? (so that it's in the right place if someone chooses to uncomment it in future to help their debugging)

Reply:
...in fact, perhaps it'd be more useful to put all these `printf` calls inside some kind of `#ifdef DEBUG` section, rather than commenting them all out entirely? \*shrug\*